easy2use

```diff
@@ -270,7 +270,7 @@ function do_k8s_local_hosts_file_minikube {
   cd ${bundles_path}/$bundle
   minikube_ip=$(minikube ip)
   echo "Update your local '../etc/hosts' file with:"
-  source ${CLI_ROOT}/utilities/K8S/get_k8s_ingress.sh | awk -v env_var="$minikube_ip" '{if (NR!=1) {print env_var" "$2}}'
+  source ${CLI_ROOT}/utilities/K8S/get_k8s_ingress.sh | awk -v env_var="$minikube_ip" '{if (NR!=1) {print env_var" "$3}}'
```

Comment:
Looking at line 797 i couldn't see the third argument. Where do we get it from? Would also be good to explain what the parameter is as have been done on line 268-269

Reply:
The `do_k8s_local_hosts_file_minikube` function will get us the list of eiffel ingress host entries to be updated in local hosts file. 

However, current `easy2use` script give below results for `./easy2use generate-local-hosts-file-minikube Eiffel`
`** Easy2Use Target Configuration **
Target Type: Kubernetes
HELM Version: V3
Domainname: mylocalkube
Cluster configfile: /home/infra/.kube/config
Cluster context: minikube
Update your local '../etc/hosts' file with:
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
192.168.49.2 nginx
`
Above result doesn't help to access the eiffel services ingress host entries from the local machine. Hence, i changed the lookup positional argument from $2 to $3 in `do_k8s_local_hosts_file_minikube` function, as it is taking the ingress values from `${CLI_ROOT}/utilities/K8S/get_k8s_ingress.sh` script.

Expected result will look like this for `./easy2use generate-local-hosts-file-minikube Eiffel,`
`** Easy2Use Target Configuration **
Target Type: Kubernetes
HELM Version: V3
Domainname: mylocalkube
Cluster configfile: /home/infra/.kube/config
Cluster context: minikube
Update your local '../etc/hosts' file with:
192.168.49.2 dummy-er-eiffel.mylocalkube
192.168.49.2 eiffel-backend-allevents-eiffel.mylocalkube
192.168.49.2 eiffel-backend-artifact-eiffel.mylocalkube
192.168.49.2 eiffel-backend-sourcechange-eiffel.mylocalkube
192.168.49.2 eiffel-backend-testexecution-eiffel.mylocalkube
192.168.49.2 eiffel-er-eiffel.mylocalkube
192.168.49.2 eiffel-frontend-eiffel.mylocalkube
192.168.49.2 eiffel-jenkins-eiffel.mylocalkube
192.168.49.2 eiffel-jenkins-fem-eiffel.mylocalkube
192.168.49.2 eiffel-nexus3-eiffel.mylocalkube
192.168.49.2 eiffel-rabbitmq-eiffel.mylocalkube
192.168.49.2 eiffel-remrem-generate-eiffel.mylocalkube
192.168.49.2 eiffel-remrem-publish-eiffel.mylocalkube
192.168.49.2 eiffel-vici-eiffel.mylocalkube
`
So, above entries can be added into local hosts file to easily access the Eiffel services hosted in k8s.

Reply:
aha, it was $3 in the awk command not the call to `do_k8s_local_hosts_file_minikube`